recipes/ufal.udpipe/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ufal.udpipe" %}
+{% set version = "1.2.0.1" %}
+{% set sha256 = "e24ebb712232d657900aee2b2e25e7ab45726089796dee7a376faf22e619eeb7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Please add the following to the `build` section here.

```yaml
  skip: true  # [win and vc<14]
```

This should drop the Windows builds where there is not a new enough compiler.

Reply:
Done. Thanks!